README.md

```diff
@@ -6,9 +6,7 @@ A utility for monitoring services
 
 Monitored is a wrapper function that writes success/error logs and [StatsD](https://github.com/statsd/statsd) metrics (gague, increment, timing) after execution. It supports both asynchronous and synchronous functions.
 
-
-[![Dependency Status][david-image]][david-url]
-[![DevDependency Status][david-dev-image]][david-dev-url]
```

Comment:
There are still URLs defined somewhere in the file (`david{,-dev}-{image,url}`) that should be removed

Reply:
Fixed @gioragutt - thanks!